pkg/k8s/kubectl.go

```diff
@@ -26,6 +26,11 @@ type kubectl struct {
 	proxyPort int
 }
 
+type VersionInfo struct {
```

Comment:
I like having a tyope to aggregate this 😍 

Two thoughts:
1) Does it need to be public? (i.e. does it need to start with a capital letter)
2) It seems like we already have [a public type with this name](https://github.com/runconduit/conduit/blob/fd3cfcb5d952b2c816beb90886df2b98c2895085/controller/gen/public/api.pb.go#L531). Even if packages are different, maybe we should choose a different name here?